pkg/image/importer/importer_test.go

```diff
@@ -0,0 +1,19 @@
+package importer
+
+import (
+	"net/http"
+	"testing"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	"github.com/openshift/origin/pkg/image/api"
+)
+
+func TestImportNothing(t *testing.T) {
```

Comment:
Wait, 1000 lines of code, the heart of this pull, and it gets a single unit test??  Is there some other spot where I see the functions (which aren't small themselves) tested?